packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -821,7 +821,16 @@ export function createLodestarMetrics(
         name: "validator_monitor_validators",
         help: "Count of validators that are specifically monitored by this beacon node",
       }),
-
+      validatorStatuses: register.gauge<"status">({
```

Comment:
we could just remove the `validatorsConnected` actually since it would require a greater refactor to collect statuses and balances in scrapeMetrics. I had left it just for backwards compatibility.

Reply:
The `validatorConnected` stat is useful for variety of other calculations and metrics so I don't think it's wise to remove it.

Reply:
yeah, I don't think it is a good idea to remove it, would be more about adding the label to that metric instead but should be fine to introduce a separate metric